modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -372,7 +372,9 @@ public String extractIndexableAttributes(
 
 				Serializable value = field.getValue(locale);
 
-				if (value == null) {
+				if ((value == null) ||
+					Objects.equals(value, StringPool.BLANK)) {
```

Comment:
@ericyanLr @noornajj why don't you use `Validator.isBlank(value)` instead of `Objects.equals(value, StringPool.BLANK)`?

Reply:
Hi @carolmariaabb 
Because value is serializable and isBlank() takes in a string value

Reply:
@noornajj can we change to `Validator.isBlank(String.valueOf(value))`?

Reply:
Sure @carolmariaabb I can change it to that

Reply:
Hello @carolmariaabb 
Should I just make this change and resend the PR?

Reply:
Yes @noornajj

Reply:
@carolmariaabb new PR sent https://github.com/liferay-forms/liferay-portal/pull/2127